src/services/authService.ts

```diff
@@ -0,0 +1,43 @@
+'use server'
+import { Session, SignInDTO } from '@/types/Auth'
+import path from 'path'
+import fs from 'fs'
+import { randomUUID, UUID } from 'crypto'
+import { User } from '@/types/User'
+import { verifyHash, verifySession } from '@/utils/auth'
+
+import { patchUser } from './userService'
+
+export const signIn = async (formData: SignInDTO): Promise<UUID> => {
+    const { login, password } = formData
+    const filePath = path.join(process.cwd(), 'src', 'data', 'users', login, 'user.json')
+    let data
+    try {
+        data = fs.readFileSync(filePath, 'utf8')
+    } catch {
+        throw new Error('User with login: ' + login + ' does not exist')
```

Comment:
W JavaScripcie używa się w takich sytuacjach template stringów:
```suggestion
        throw new Error(`User with login: ${login} does not exist`)
```

Reply:
say {ok}